build.gradle.kts

```diff
@@ -13,7 +13,33 @@ dependencies {
     implementation(kotlin("stdlib-jdk8"))
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.20.0")
     implementation("org.joy.prime:joy-data-kotlin:0.0.1")
+    implementation("parts.wisdom.djlkotlin:api:0.0.1")
+    implementation("parts.wisdom.djlkotlin:basicdataset:0.0.1")
+    implementation("parts.wisdom.djlkotlin:model-zoo:0.0.1")
+    implementation("org.slf4j:slf4j-simple:1.7.30")
     testImplementation("org.junit.jupiter:junit-jupiter:5.6.2")
+
+    val djlVersion: String by project
+    val djlEngine: String by project
+    when (djlEngine.toLowerCase()) {
```

Comment:
I was picturing the underlying `ai.djl` libraries as being reexported by `parts.wisdom.djlkotlin`, so that a client project like `ARC-raider` would only need to declare dependency on the `parts.wisdom.djlkotlin` libraries that it needs. Thoughts on that?

Reply:
Not sure I totally follow. Are you talking about this engine code specifically ie mxnet vs pytorch config? Or all the ai.djl code. IE are you suggesting we write wrapper functions / objects to completely hide the underlying ai.djl functionality?

Reply:
Now that I re-read your PR, my concern is narrower than I realized when I wrote the original comment. Also, I realize you are doing *exactly* what I did in my Mnist example :-), so I'm objecting to my own code.

My concern is with the client of `parts.wisdom.djkotlin` doing its own direct import of the engine. For now, at least, this is risky because the engine version numbers have to correspond (in a complex way) to the wrapper library version number. I'm wishing that instead the client could just import appropriate `parts.wisdom.djkotlin` packages and then would have everything it needs. But again, you are doing exactly what I did in my example -- even if you agree with my point, we don't have to fix it now.

> IE are you suggesting we write wrapper functions / objects to completely hide the underlying ai.djl functionality?
Definitely not that!

I've been thinking of the `parts.wisdom.djlkotlin.xyz` packages as being a Kotlin-programmer's-eye view of the DJL. By importing `parts.wisdom.djlkotlin.xyz`, you get some Java stuff and some Kotlin stuff for xyz -- the api, the model-zoo, whatever. At this point it happens to be 100% Java stuff, and 100% provided by `ai.djl.xyz`.

Reply:
Cool, updated this pr and added a pr in the djl-kotlin to remove having to import the engine here.